lib/redcord/actions.rb

```diff
@@ -24,6 +24,18 @@ def self.included(klass)
   module ClassMethods
     extend T::Sig
 
+    sig { returns(Integer) }
+    def count
+      Redcord::Base.trace(
+       'redcord_actions_class_methods_count',
+        model_name: name,
+      ) do
+        where(
+          id: Redcord::RangeInterval.new(min: 1),
```

Comment:
I would worry that this is a potentially expensive operation to do in one round trip? https://redis.io/commands/zrangebyscore

Reply:
My impression is we mostly want this for monitoring purposes, so having the id as an index set in Redis by default might not be worth it if it can cause performance issues.

Especially since we don't have vacuuming yet (working on it 😞 ), the id index set seems like it would get bloated, making this an even more expensive operation since it would go through and remove expired ids from the set as well.